test/test_preprocessor.py

```diff
@@ -104,8 +104,8 @@ def test_remove_substrings():
     preprocessor = PreProcessor(remove_substrings=["This is a header.", "wiki", "🪲"])
     documents = preprocessor.process(document)
```

Comment:
The `document` object doesn't need to be a `dict` in this test. It is actually better if it's a `Document`, it was a dict just because of this PreProcessor's limitation